### ISSUE TITLE
Avoid multiple files that are exactly the same.

### ISSUE THREAD

**Issue by tomazcunha** [zabel-xyz/local-history#156]
Every time I save the file it generates a new history, even if the file is exactly the same as the previous one. I would like an option to keep only the most recent file and delete the previous one which is exactly the same. To avoid multiple files exactly the same. Thanks for the extension.

**Comment by oswald522** [comment]
I strongly agree with this feature. The number of identical files saved each time is too high. There are so many of them that I can't even find the specific files that were modified.

**Comment by piggov** [comment]
Duplicate of #143

**Comment by vanowm** [comment]
I don't think deleting previous identical backup file is a good idea. I think it's better not create new backup file at all if previous backup file is identical.

However, if we limit this to create backup file only if content was modified before that content was saved to the original file, wouldn't this create an issue when file was modified by git?
I think since git could overwrite the file; we won't have copy of current content before it was overwritten by git.
